infrastructure/ssz/src/main/java/tech/pegasys/teku/infrastructure/ssz/collections/impl/BitvectorImpl.java

```diff
@@ -76,6 +76,16 @@ public List<Integer> getSetBitIndices() {
     return data.stream().boxed().toList();
   }
 
+  public BitvectorImpl or(final BitvectorImpl other) {
+    if (other.getSize() > getSize()) {
```

Comment:
i'd think we want to make sure the sizes match exactly if we're doing a bitwise or, otherwise we'll be needing to decide if the stragglers are left or right etc... `other.size() != getSize`

Reply:
mm yes but this mimic the `BitlistImpl` or which does the same. We may say that this is a vector so it could be more strict but I'm more on the side to have a similar behavior.

Reply:
its probably bad behaviour to mimic... you can mis-match and do bitwise or partially.... future @tbenr problem to find i guess...

Reply:
Is this feature really used somewhere?
If yes we could split it to several safer steps, but it could affect performance

Reply:
k lets be more stringent and maybe open in the future.
@zilm13 we won't (at least for 7549)

Reply:
i really do think we're better off making this one right rather than copying a bad idea - especially on a bitwise or... at least then we know why its a greater than if it causes a break...